src/workspaces/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -97,7 +98,7 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
   const addCollaborator = (collaboratorEmail) => {
     if (!validate.single(collaboratorEmail, { email: true, exclusion: aclEmails })) {
       setSearchValue('');
-      setAcl(append({ email: collaboratorEmail, accessLevel: 'READER' }));
+      setAcl(append({ email: collaboratorEmail, accessLevel: 'READER' } as AccessEntry));
```

Comment:
The value being appended did not have an explicit type, so append's type safety was failing this line.

Reply:
Can we avoid the cast by splitting out the variable declaration?

```suggestion
      const aclEntry: AccessEntry = { email: collaboratorEmail, accessLevel: 'READER' };
      setAcl(append(aclEntry));
```

Reply:
That seems like a good approach to me. I'll see if I can get that to work.

Reply:
It looks like this object is missing some attributes to a valid Access Entry.

It seems like it should have `pending`, `canShare`, and `canCompute` attributes.

Reply:
It sounds like either this line is incorrect or the type may be inaccurate (maybe those fields are actually optional).